core/src/main/scripts/importer/validateData.py

```diff
@@ -965,6 +972,12 @@ def checkHeader(self, cols):
                 'that the Uniprot canonical isoform is used when drawing Pfam '
                 'domains in the mutations view',
                 extra={'line_number': self.line_number})
+        elif not 'swissprot_identifier' in self.meta_dict:
+            self.logger.warning(
+                "A SWISSPROT column was found in a file without an "
+                "associated 'swissprot_identifier' metadatum, assuming "
+                "'swissprot_identifier: name'.",
```

Comment:
thinking about it, I think we can simplify both the code and the behaviour if we make this meta field mandatory (in the validator). This is probably the best option now that we are in a "transition phase" between old and new VEP installations...having a default value will probably just result in unintended behaviour for one of the two cases (either the people having old VEP or the ones having new VEP - assuming the difference is caused by VEP versios). Having to set a meta property is clear for both cases. 

For the loader, the default can stay. I.e. when circumventing the validation step you can expect to have some sub-optimal results.